portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/Sites.testcase

```diff
@@ -311,6 +312,80 @@ definition {
 			parentSiteName = "Site Name");
 	}
 
+	@description = "This is a test for LPS-102711. The changes of site scope configuration in Instance Settings should be applied to the site."
+	@priority = "4"
+	test ViewChangesOfSiteScopeInInstanceSettings {
```

Comment:
Had better to name it to ViewChangesOfSiteScopeConfigurationDefinedInInstanceSettings